### PR TITLE
chapter5

### PR DESCRIPTION
**Pull request by thieuhai**
ready

### CODE REVIEW

**Comment by thieuhai** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= full_title yield:title %></title>
+    <title><% =full_title yield:title %></title>
```
Comment (phucmh-1356) [inline app/views/layouts/application.html.erb]:
```suggestion
    <title><%= full_title yield(:title) %></title>
```

**Comment by thieuhai** [comment]
ready

**Comment by phucmh-1356** [comment]
commented
gộp commits rồi merged nhé

**Pull request merged**